include/tscpp/util/TsSharedMutex.h

```diff
@@ -184,8 +186,8 @@ class shared_mutex
 
   // In debug builds, make sure shared vs. exlusive locks and unlocks are properly paired.
   //
-  X(std::atomic<bool> _exclusive;)
-  X(std::atomic<int> _shared;)
+  X(std::atomic<bool> _exclusive{false};)
+  X(std::atomic<int> _shared{0};)
```

Comment:
This isn't necessary.

Reply:
According to this: https://en.cppreference.com/w/cpp/atomic/atomic/atomic

> The default constructor is trivial: no initialization takes place other than [zero initialization](https://en.cppreference.com/w/cpp/language/zero_initialization) of static and thread-local objects. [std::atomic_init](https://en.cppreference.com/w/cpp/atomic/atomic_init) may be used to complete initialization.

I also ran into an uninitialized value during local testing, which drove this change.

Reply:
Oops, sorry, I thought the std::atomic constructor always zero initialized, even for stack instances.

Reply:
No problem!